src/pages/task/taskTabs/buildBaronAndAnnotations/JiraTicketList/index.tsx

```diff
@@ -0,0 +1,16 @@
+import { CreatedTicketsQuery } from "gql/generated/types";
+import JiraTicketRow from "./JiraTicketRow";
+
+type CreatedTickets = CreatedTicketsQuery["bbGetCreatedTickets"];
+
+const JiraTicketList: React.FC<{
```

Comment:
[nit] It may be nice to combine the whole list into one component since this wrapper is so simple 🤷‍♀️ but up to you!